### DISCUSSION TITLE
Using with load balancer and multiple servers

### DISCUSSION THREAD

**Discussion by MaTToX3** [Mimetis/Dotmim.Sync#1028] [General]
Hello!

I have a question; how would one set up Dotmim.Sync when the web app uses a load balancer and two servers. 

We have such scenario in one of our projects and we did the basic setup. When syncing, we are getting a lot of session loss/batchinfopartnotfound exceptions. We suspect that load balancer and multiple servers are an issue. 

So, I am wondering the following:
- can Dotmim.Sync even support such a scenario?
- if yes, how to properly set it up?
- if not, can we do something to make it work (I dunno, custom memory cache implementation, setting a path for a server, where temporary files are uploaded)?

I hope I explained the challenges we face well enough. 

Any answer would be greatly appreciated.

**Comment by Mimetis** [comment]
You need a session affinity on your load balancer if you want to scale on multiple servers.
A sync involve multiple Https calls that need to be tight to a session on the server.

Otherwise, you can also use a different session manager, like REDIS I guess (that being said, I never tested it)
